### ISSUE TITLE
Deprecated transferMarket

### ISSUE THREAD

**Issue by cyupa** [sorare/api#485]
Hello and thanks for publishing and maintaining this API.

I was looking into picking up the latest card/token auctions and the old way of doing it was via `transferMarket` which now seems to be deprecated and redirecting us to use the `tokens` root.

The problem with the `auctions` query on the tokens root is that it requires auction id which I wouldn't be able to get as I would with `englishAuctions(last: 10)` for example.

What would be the way to go about this?
Thanks!

**Comment by redox** [comment]
> The problem with the auctions query on the tokens root is that it requires auction id which I wouldn't be able to get as I would with englishAuctions(last: 10) for example.

Agreed, spot this as well. We need to add the equivalent in the tokens root.
